lib/instruments.js

```diff
@@ -395,14 +395,20 @@ Instruments.prototype.spawnInstruments = function () {
     env.DYLD_INSERT_LIBRARIES = path.resolve(iwdPath, "InstrumentsShim.dylib");
     env.LIB_PATH = iwdPath;
   }
-  var logArgs = [this.instrumentsPath].concat(_.clone(args));
-  logArgs = _.map(logArgs, function (arg) {
+  var instrumentsExecArgs = [this.instrumentsPath].concat(_.clone(args));
+  instrumentsExecArgs = _.map(instrumentsExecArgs, function (arg) {
+    if (arg === null) {
```

Comment:
this should ultimately be handled by our spawn/exec wrapper so it happens in just one place